src/api/index.js

```diff
@@ -49,4 +50,17 @@ const getProfileData = authState => {
   }
 };
 
-export { sleep, getExampleData, getProfileData, getDSData };
+const submitForm = intakeObj => {
+  try {
+    return axios.post(apiFormUrl, intakeObj).then(res => {
+      console.log(res.data);
+    });
+  } catch (error) {
+    return new Promise(() => {
+      console.log(error);
+      return [];
```

Comment:
why are you returning an empty array in the catch? Is that what they want for an error?

Reply:
Yes